source/views/transportation/bus/map.js

```diff
@@ -60,13 +61,21 @@ export class BusMapView extends React.PureComponent {
     },
   }
 
-  onRegionChange = (region: {
+  onRegionChangeComplete = (region: {
     latitude: number,
     latitudeDelta: number,
     longitude: number,
     longitudeDelta: number,
   }) => {
-    this.setState(() => ({region}))
+    this.setState(state => {
+      const initialRegion = state.initialRegion
```

Comment:
Don't forget to rename it in here too. I might call the parameter "newRegion" so that you can keep this block as "region".